### PR TITLE
Add signmessage

### PR DESCRIPTION
**Pull request by LeoComandini**

### CODE REVIEW

File: green_cli/common.py [hunk 1]
```diff
@@ -535,6 +535,16 @@ def bumpfee(session, previous_txid, fee_multiplier, subaccount, wait, timeout):
...
     return _send_transaction(session, details, wait, timeout)
 
+@green.command()
+@click.argument('address', type=str, expose_value=False, callback=details_json)
+@click.argument('message', type=str, expose_value=False, callback=details_json)
+@with_login
+@print_result
+def signmessage(session, details):
+    """Sign a message"""
+    auth_handler = gdk.sign_message(session.session_obj, json.dumps(details))
```
Comment (jkauffman1) [inline green_cli/common.py:545]: Is there a reason we didn't use `@with_gdk_resolve`?

Comment (LeoComandini) [reply green_cli/common.py]: no, changed to use it

**Comment by jgriffiths** [comment]
utack ba5ce05f3e6d1917e7b16598659365f03a241048

**Comment by jkauffman1** [comment]
utack ba5ce05f3e6d1917e7b16598659365f03a241048

**Pull request closed without merging**